### PR TITLE
Update react-nodejs example to use concurrently to start two servers

### PR DESCRIPTION
**Pull request by matv-stripe**
Problem:
@deven367 is not able to run the sample app https://github.com/markdoc/markdoc/discussions/23#discussioncomment-2751894

Solution
I think it might potentially be a problem with Windows. I think switching to use `concurrently` might help solve this

### CODE REVIEW

**Review by mfix-stripe — CHANGES_REQUESTED** [review]
Can we remove the package-lock.json file here?

File: examples/react-nodejs/package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "scripts": {
     "start:client": "react-scripts start",
     "start:server": "node server.js",
+    "start": "concurrently \"yarn start:client\" \"yarn start:server\"",
```
[the remark this answers is not in the document]
Comment (matv-stripe) [reply examples/react-nodejs/package.json:20]: forgot to push that commit, thanks for catching!

**Comment by matv-stripe** [comment]
ptal @mfix-stripe

**Review by mfix-stripe — APPROVED** [review]

**Pull request merged**